docs/subcommands/CLI Setup/logout.md

```diff
@@ -26,7 +26,7 @@ If any such fields are found, their values will be replaced with an empty string
 AUTHENTICATION ORDER
 ionosctl uses a layered approach for authentication, prioritizing sources in this order:
   1. Global flags
-  2. Environment variables
+  2. Environment variables credentials test failed
```

Comment:
Did  this generate like this?

`credentials test failed`

Reply:
seems so